### ISSUE TITLE
"can't modify frozen String" with bundler-1.11.2

### ISSUE THREAD

**Issue by AntonTrapp** [bundler/bundler#4459]
ruby-2.3.0/bundler-1.11.2

```ruby
...bundler/source/rubygems.rb:65:in `block in to_lock': can't modify frozen String (RuntimeError)
```

when running with ```export RUBYOPT=--enable-frozen-string-literal```

**Comment by segiddins** [comment]
What rubygems version are you running?

**Comment by AntonTrapp** [comment]
gem: 2.5.1 (used with rvm: 1.27.0 in own gemset)

**Comment by segiddins** [comment]
This should be fixed with bundler 1.12

**Comment by segiddins** [comment]
Closing as presumably fixed
